package.json

```diff
@@ -1,48 +1,54 @@
 {
-  "name": "@signalk/freeboard-sk",
-  "version": "0.0.4",
-  "description": "Openlayers chartplotter implementation for Signal K",
```

Comment:
Why no description? This is shown in server App store.

Reply:
NPM package is the group of files in `dist/freeboard` in which teh package.json is:
```
{
    "name": "@signalk/freeboard-sk",
    "version": "0.1.0",
    "description": "Openlayers chartplotter implementation for Signal K",
    "keywords": [
        "signalk-webapp"
    ],
    "repository": "https://github.com/SignalK/freeboard-sk",
    "scripts": {
        "format": "prettier-standard 'src/**/*.js'",
        "test": "mocha"
    },
    "author": "robert@42.co.nz",
    "contributors": [
        { "name": "panaaj@hotmail.com" }
    ],
    "license": "Apache-v2",
    "dependencies": {
    },
    "devDependencies": {
    "mocha": "^5.0.5",
    "prettier-standard": "^8.0.0"
    }
}
```